challenges/lib/src/test/java/challenges/utilities/AnimalShelterTest.java

```diff
@@ -0,0 +1,28 @@
+package challenges.utilities;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class AnimalShelterTest {
+    @Test
+    public void animalShelter() throws Exception{
+        AnimalShelter animalShelter = new AnimalShelter();
+        animalShelter.enqueue(new Dog("dog1"));
+        animalShelter.enqueue(new Dog("dog2"));
+        animalShelter.enqueue(new Dog("dog3"));
+        animalShelter.enqueue(new Cat("cat1"));
+        animalShelter.enqueue(new Cat("cat2"));
+        animalShelter.enqueue(new Cat("cat3"));
+        Object expected1 = "dog1";
+        Object actual1 = animalShelter.dogs;
+        assertEquals(expected1, actual1);
```

Comment:
Line 18 you are comparing the string of dog1 to the class of animalShelter.dogs (which is a Queue). Which results in the test failing. You should start by enqueuing 1 animal for proof it works. Then some kind of assertion like assertEqual("dog1", dogs.Peek()).